guides/source/action_view_helpers.md

```diff
@@ -73,6 +73,24 @@ Returns an HTML image tag for the source. The source can be a full path or a fil
 image_tag("icon.png") # => <img src="/assets/icon.png" />
 ```
 
+#### picture_tag
+
+Returns an HTML picture tag for the source. It supports passing a String, an Array or a Block.
+
+```ruby
+picture_tag("icon.webp", "icon.png")
+```
+
+This generates the following HTML:
+
+```html
+<picture>
+  <source srcset="/assets/icon.webp" type="image/webp" />
+  <source srcset="/assets/icon.png" type="image/png" />
+  <img src="/assets/icon.png" />
+</picture>
+```
```

Comment:
Could you please move this method in an alphabetical order.

Reply:
done @akhilgkrishnan